exercises/Hangman Game/Hangman.py

```diff
@@ -0,0 +1,54 @@
+# Variablen
+from operator import ge
+
+
+gesucht = 'test'
+
+gefunden = []
+falsch_geraten = []
+
+def show():
+    print('Falsche Buchstaben:', falsch_geraten)
+    for buchstabe in gesucht:
+        if buchstabe in gefunden:
+            print(buchstabe, end=' ')
+    else:
+        print('_', end=' ')
```

Comment:
Das `else` gehört zum `if` auf Z12, muss daher auch gleich weit eingerückt sein.

```suggestion
    for buchstabe in gesucht:
        if buchstabe in gefunden:
            print(buchstabe, end=' ')
        else:
            print('_', end=' ')
```